aspnetcore/host-and-deploy/web-farm.md

```diff
@@ -34,33 +41,34 @@ The [ASP.NET Core Data Protection system](xref:security/data-protection/introduc
 | -------- | ------------------- |
-| Cookie authentication | Data Protection (encrypted cookies) (see <xref:security/data-protection/configuration/overview>).<br><br>For more information, see <xref:security/authentication/cookie>. |
-| Identity | Data Protection (encrypted cookies) (see <xref:security/data-protection/configuration/overview>) and database configuration.<br><br>For more information, see <xref:security/authentication/identity>. |
-| Session | Data Protection (encrypted cookies) (see <xref:security/data-protection/configuration/overview>) and Caching (see <xref:performance/caching/distributed> and <xref:performance/caching/response>).<br><br>For more information, see [Session and app state: Session state](xref:fundamentals/app-state#session-state). |
+| Authentication | Data Protection (see <xref:security/data-protection/configuration/overview>).<br><br>For more information, see <xref:security/authentication/cookie> and <xref:security/cookie-sharing>. |
+| Identity | Authentication and database configuration.<br><br>For more information, see <xref:security/authentication/identity>. |
+| Session | Data Protection (encrypted cookies) (see <xref:security/data-protection/configuration/overview>) and Caching (see <xref:performance/caching/distributed>).<br><br>For more information, see [Session and app state: Session state](xref:fundamentals/app-state#session-state). |
 | TempData | Data Protection (encrypted cookies) (see <xref:security/data-protection/configuration/overview>) or Session (see [Session and app state: Session state](xref:fundamentals/app-state#session-state)).<br><br>For more information, see [Session and app state: TempData](xref:fundamentals/app-state#tempdata). |
-| Anti-forgery | Data Protection (encrypted cookies) (see <xref:security/data-protection/configuration/overview>).<br><br>For more information, see <xref:security/anti-request-forgery>. |
+| Anti-forgery | Data Protection (see <xref:security/data-protection/configuration/overview>).<br><br>For more information, see <xref:security/anti-request-forgery>. |
+| App data | When an app is scaled to multiple instances, there might be app state that requires sharing across nodes. If the state is transient, consider sharing the state by implementing [IDistributedCache](/dotnet/api/microsoft.extensions.caching.distributed.idistributedcache). If the shared state requires persistence, consider storing the shared state in a database. |
 
 ## Troubleshoot
 
-When Data Protection or Caching isn't configured for a web farm environment, intermittent errors occur when requests are processed. This occurs because nodes don't share the same resources (or user requests aren't always routed back to the same node).
+When Data Protection or Caching isn't configured for a web farm environment, intermittent errors occur when requests are processed. This occurs because nodes don't share the same resources and user (or user requests aren't always routed back to the same node).
```

Comment:
"...the same resources and user requests aren't always routed back to the same node."